### DISCUSSION TITLE
tutorials/5-uprobe-bashreadline/

### DISCUSSION THREAD

**Discussion by giscus** [eunomia-bpf/bpf-developer-tutorial#147] [General]
# tutorials/5-uprobe-bashreadline/

Unlock the potential of eBPF

https://eunomia.dev/tutorials/5-uprobe-bashreadline/

**Comment by MuGeeee** [comment]
Using the bash_readline function with ebpf attach to uprobe does not capture the first command typed in the shell. Has anyone had a similar problem?

**Comment by vigneshs-dev** [comment]
This looks nice. I can get the right and wrong commands which I put in the bash shell.
